lib/wibox/widget/slider.lua

```diff
@@ -318,6 +319,7 @@ function slider:set_value(value)
     if changed then
         self:emit_signal( "property::value", value)
         self:emit_signal( "widget::redraw_needed" )
+        self:emit_signal("widget::layout_changed")
```

Comment:
Please add an `if self._private.handle_widget then` around this.

You also want here is to create a custom setter for `set_handle_widget` instead of using the standard ones in the loop at line 292 so it can have the signal too (copy/paste the boilerplate code).

Reply:
Done I think